docs/apis/programming_guide.md

```diff
@@ -283,7 +284,7 @@ This will create a new DataSet by converting every String in the original
 set to an Integer. For more information and a list of all the transformations,
 please refer to [Transformations](#transformations).
 
-Once you have a DataSet that needs to be written to disk you call one
+Once you have a DataSet that needs to be processed further you can call one
```

Comment:
I think I understand why you changes this to "processed further", but it does not make sense with print(). ;)

Maybe: once you want to write or read a result?